biz.aQute.resolve/src/biz/aQute/resolve/DistroRepository.java

```diff
@@ -0,0 +1,21 @@
+package biz.aQute.resolve;
+
+import java.io.File;
+
+import org.osgi.resource.Resource;
+
+import aQute.bnd.osgi.Domain;
+import aQute.bnd.osgi.repository.ResourcesRepository;
+import aQute.bnd.osgi.resource.ResourceBuilder;
+
+public class DistroRepository extends ResourcesRepository {
```

Comment:
So I think that my implementation needs to be reworked, but in general, would it make sense for us to allow for the case where distro's could be augmentable?